src/GraphQLParser/Visitors/PrintContextExtensions.cs

```diff
@@ -21,7 +21,7 @@ public static ValueTask WriteAsync<TContext>(this TContext context, ROM value)
         var task =
 #if NETSTANDARD2_0
         // no cancellationToken support on netstandard2.0
-        context.Writer.WriteAsync(value.ToString());
+        context.Writer.WriteAsync(value.ToString()); //ISSUE: allocation - either WriteAsync(value.ToString()) or Write(char value) in a loop
```

Comment:
Calling `Write(char value)` in a loop would be a performance killer.  Much better to have an allocation if needed.

Reply:
ok